oletus.mjs

```diff
@@ -1,6 +1,8 @@
 import assert, { AssertionError } from 'assert'
 import { stripCwd } from './util.mjs'
 
+let incremental = 0
+
```

Comment:
We're keeping an incremental global ID per process, allowing the test runner to correlate pending tests with completed tests.